stock_manual_transfer/views/stock_manual_transfer_views.xml

```diff
@@ -0,0 +1,111 @@
+                    </group>
+                    <notebook>
+                        <page string="Transfer Lines" name="transfer_lines">
+                            <field name="transfer_line_ids" nolabel="1" readonly="state != 'draft'">
+                                <list editable="bottom">
+                                    <field name="sequence" widget="handle" />
+                                    <field name="product_id" />
+                                    <field name="product_uom_qty" />
+                                    <field name="product_uom_id" column_invisible="1" />
+                                    <field name="product_uom_id" groups="uom.group_uom" optional="show" />
+                                    <field name="product_uom_category_id" column_invisible="True" />
+                                </list>
+                                <form>
+                                    <group>
+                                        <group>
+                                            <field name="product_id" />
+                                            <field name="product_uom_qty" />
+                                            <field name="product_uom_id" groups="uom.group_uom" />
+                                            <field name="product_uom_category_id" column_invisible="True" />
+                                            <field name="sequence" />
+                                        </group>
+                                    </group>
+                                </form>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+                <chatter />
+            </form>
+        </field>
+    </record>
+
+    <record id="stock_manual_transfer_tree_view" model="ir.ui.view">
+        <field name="name">stock.manual_transfer.list</field>
+        <field name="model">stock.manual_transfer</field>
+        <field name="arch" type="xml">
+            <list>
+                <field name="name" />
+                <field name="warehouse_id" readonly="state != 'draft'" />
+                <field name="date_planned" readonly="state != 'draft'" />
+                <field name="route_id" readonly="state != 'draft'" />
+                <field name="state" />
+            </list>
+        </field>
+    </record>
+
+    <record id="stock_manual_transfer_action" model="ir.actions.act_window">
+        <field name="name">Manual Transfer</field>
+        <field name="path">stock_manual_transfer</field>
+        <field name="res_model">stock.manual_transfer</field>
+        <field name="help" type="html">
+          <p class="oe_view_nocontent_create">
+            Click to add a new manual transfer.
+          </p>
+        </field>
+    </record>
+
+    <menuitem
+        id="menu_stock_manual_transfer"
+        name="Internal Manual Transfer"
```

Comment:
Shouldn't it be in plural?

If so, please fix window action name. I think this is an issue comming from the previous version.

Reply:
I do not know, you tell me hahaha, so should it be fixed on the previous version?

Reply:
Yes please, that was probably my bad.

Reply:
Done

Reply:
For the record, it was done in #1693